pywbem_mock/_mainprovider.py

```diff
@@ -100,25 +105,16 @@ def __init__(self, conn, datastore, default_namespace=DEFAULT_NAMESPACE):
               variables are extracted for the repository including
               the host and disable_pull_operations attributes.
 
-            datastore(:class:`~pywbem_mock.InMemoryRepository` or other subclass of `~pywbem_mock.baseRepository`):
-              class defining the data store for the CIM repository.
-
-            default_namespace(:term:`string`):
-              The default namespace defined if no namespace is specified
-              on a request.
-              TODO: Not sure this is necessary. On server side there should
-              be namespace with every request.
-        """  # noqa: E501
-
+            datastore(:class:`~pywbem_mock.InMemoryRepository` or other subclass of `~pywbem_mock.baseRepository`):  # noqa: E501
+             class defining the data store for the CIM repository.
```

Comment:
I suggest to remove any mentioning of `InMemoryRepository` from the description of `MainProvider`, and use only `BaseRepository`. Occurs several times.

Reply:
All references changed to BaseRepository